docs/content/connectors/mysql-cdc.md

```diff
@@ -29,7 +29,7 @@ In order to setup the MySQL CDC connector, the following table provides dependen
 
 ```Download link is available only for stable releases.```
 
-Download [flink-sql-connector-mysql-cdc-2.5-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mysql-cdc/2.5-SNAPSHOT/flink-sql-connector-mysql-cdc-2.5-SNAPSHOT.jar) and put it under `<FLINK_HOME>/lib/`.
+Download flink-sql-connector-mysql-cdc-2.5-SNAPSHOT.jar and put it under `<FLINK_HOME>/lib/`.
```

Comment:
I'm a bit concerned about the word "download" here, as there isn't any place from which users could "download" the snapshot JAR, other than building it locally from source code. Furthermore, is there any specific reason that user must choose the snapshot JAR instead of a stable release here?